apps/server/src/services/vcProviders/sophtronVc.test.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  sophtronVcAccountsData,
+  sophtronVcIdentityData,
+  sophtronVcTranscationsData
+} from '../../test/testData/sophtronVcData'
+import getVC from './sophtronVc'
+
+describe('getVc', () => {
+  const connectionId = 'connectionId'
+  const userId = 'userId'
+  const accountId = 'accountId'
+  const startTime = '1/1/2024'
+  const endTime = '6/1/2024'
```

Comment:
Is having hard-coded dates a concern? Will these dates work years down the road?

Reply:
They're parameters being passed to an api that is being mocked, so they could actually be the word junk and it the tests would still pass.